src/utils/constants.js

```diff
@@ -5,6 +5,8 @@ export const STATUS_ERROR = 'ERROR';
 export const DEFAULT_FETCHER_UPDATE_INTERVAL = 5000;
 export const DEFAULT_TIMEOUT_BEFORE_FILE_DELETION = 0;
 
+export const DEFAULT_TIMEOUT_BEFORE_JOB_DELETION = 10000;
```

Comment:
Can this const be also used in `src/routes/JobProfile` file?

Reply:
done